.azure/pipelines/ci.yml

```diff
@@ -431,6 +431,9 @@ stages:
       - name: Windows_Test_Results
         path: artifacts/TestResults/
         publishOnError: true
+      - name: Windows_Test_Dumps
+        path: artifacts/dumps/
+        publishOnError: true
```

Comment:
Suggest making the dump output location artifacts/log and reverting changes to this file. That's where dump_process.ps1 output already goes.

Reply:
Dumps are not logs, id rather keep them separate.